library/core/src/ptr/const_ptr.rs

```diff
@@ -520,9 +520,15 @@ impl<T: ?Sized> *const T {
     /// Calculates the distance between two pointers. The returned value is in
     /// units of T: the distance in bytes is divided by `mem::size_of::<T>()`.
     ///
-    /// This function is the inverse of [`offset`].
+    /// This is equivalent to `(self as isize - other as isize) / mem::size_of::<T>()`.
     ///
-    /// [`offset`]: #method.offset
```

Comment:
NB: I dropped the note that it was the "inverse" of offset because this is a genuinely confusing statement.

`offset` is an asymmetric binary operation, it's like talking about the "inverse" of x<sup>y</sup>. There are two answers! (yth root and log<sub>x</sub>)